torch/testing/_creation.py

```diff
@@ -135,53 +153,72 @@ def clamp(a, l, h):
         shape = shape[0]  # type: ignore[assignment]
     shape = cast(Tuple[int, ...], tuple(shape))
 
+    if noncontiguous and memory_format is not None:
+        raise ValueError(
+            f"The parameters `noncontiguous` and `memory_format` are mutually exclusive, "
+            f"but got {noncontiguous=} and {memory_format=}"
```

Comment:
I didn't know writing `noncontiguous=` like that would work.

Is it equivalent to `f"noncontiguous={noncontiguous}"`?

Reply:
Yes, that is equivalent. This is an addition to Python 3.8, which is the lowest supported version for PyTorch as of today. See https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging